apps/studio/components/interfaces/UserDropdown.tsx

```diff
@@ -0,0 +1,146 @@
+import { ProfileImage } from 'components/ui/ProfileImage'
+import { useSignOut } from 'lib/auth'
+import { IS_PLATFORM } from 'lib/constants'
+import { useProfile } from 'lib/profile'
+import { Command, FlaskConical, Settings } from 'lucide-react'
+import { useTheme } from 'next-themes'
+import Link from 'next/link'
+import { useRouter } from 'next/navigation'
+import { useAppStateSnapshot } from 'state/app-state'
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuGroup,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuRadioGroup,
+  DropdownMenuRadioItem,
+  DropdownMenuSeparator,
+  DropdownMenuTrigger,
+  SidebarMenuButton,
+  Theme,
+  cn,
+  singleThemes,
+} from 'ui'
+import { useSetCommandMenuOpen } from 'ui-patterns/CommandMenu'
+
+export function UserDropdown() {
+  const { profile } = useProfile()
+  const appStateSnapshot = useAppStateSnapshot()
+  const { theme, setTheme } = useTheme()
+  const signOut = useSignOut()
+  const router = useRouter()
+
+  const setCommandMenuOpen = useSetCommandMenuOpen()
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <SidebarMenuButton
+          className={cn(
+            'text-sm',
+            'group-data-[state=expanded]:h-10',
+            'p-0.5 group-data-[state=expanded]:pr-2 group-data-[state=expanded]:pl-1'
+          )}
+          size={'default'}
+          hasIcon={false}
+          asChild
+          isActive={false}
+        >
+          <button>
+            <div className="aspect-square h-7 w-7 rounded-md border flex-shrink-0">
+              <ProfileImage
+                alt={profile?.username}
+                src={profile?.profileImageUrl}
+                className="w-7 h-7 rounded-md"
+              />
+            </div>
+            <span className="flex flex-col gap-0">
+              {profile?.username}
+              <span className="text-foreground-lighter text-xs">{profile?.primary_email}</span>
+            </span>
```

Comment:
thinking we should use a different UI here for local set up?
![image](https://github.com/user-attachments/assets/7fab1dce-79d3-4b63-8014-b204062c1e73)

Reply:
oh, we could just hide it I suppose

Reply:
should fix it?
https://github.com/supabase/supabase/commit/843f7437f263115fd57a80defe4feb3b2186316a

Reply:
Fixed now, we have an alternative dropdown for local and self hosted.
thanks @joshenlim for flaggin!